### DISCUSSION TITLE
character didn't move

### DISCUSSION THREAD

**Discussion by coolkid8715** [bevyengine/bevy#6943] [Q&A] [answered]
I try fix this problem but I can't still move character.
I can't find any suspicious codes on my eyes

[space.zip](https://github.com/bevyengine/bevy/files/10218387/space.zip)

PS: I made these codes based on this [video](https://youtu.be/j7qHwb7geIM) and this [cheat book](https://bevy-cheatbook.github.io/#unofficial-bevy-cheat-book)

**Comment by tim-blackbird** [comment, marked as answer]
Looks like `player_keyboard_event_system` is never added to the app.

**Reply by coolkid8715** [reply]
![image](https://user-images.githubusercontent.com/87269142/207509865-d47ff7c3-3525-4d58-97e7-f48bdb0635be.png)
ok but sprite is.... I will fix this, Thank you!

**Reply by tim-blackbird** [reply]
The sprite is being spawned every frame, use `add_startup_system` to add a system that only runs once.

**Reply by coolkid8715** [reply]
Thank you very much!

**Reply by coolkid8715** [reply]
> The sprite is being spawned every frame, use `add_startup_system` to add a system that only runs once.

I fix that situation. I don't know reason add_startup_system is make game crash for player system. 
so I use "add_startup_system_to_stage"
